python/ray/tune/analysis/experiment_analysis.py

```diff
@@ -183,6 +183,82 @@ def __init__(self, experiment_checkpoint_path, trials=None):
         super(ExperimentAnalysis, self).__init__(
             os.path.dirname(experiment_checkpoint_path))
 
+    def get_best_trial(self, metric, mode="max", scope="all"):
+        """Retrieve the best trial object. Compare all trials' scores on
+            `metric`, based on `mode`. Only factor in `scope` steps.
+
+        Args:
+            metric (str): Key for trial info to order on.
+            mode (str): One of [min, max].
+            scope (str): One of [all, last].
+        """
+        if mode not in ["max", "min"]:
+            logger.warning(
+                "ExperimentAnalysis: attempting to get best trial for "
+                "metric {} for mode {} not in [\"max\", \"min\"]".format(
+                    metric, mode))
+            return None
+        if scope not in ["all", "last"]:
```

Comment:
might have missed it but where is `scope` used?

also does `scope=last` mean literally only the last step? prob mention that in the documentation

Reply:
yeah scope=last means literally only the last step.